Snakefile

```diff
@@ -420,92 +602,1307 @@ rule atlas_experiment_summary:
+            [ -e "{wildcards.accession}_${{p}}-normalized-expressions.tsv.decorated.tmp" ] && rm "{wildcards.accession}_${{p}}-normalized-expressions.tsv.decorated.tmp"
+
+        done
+        touch {output}   
+        """   
+
+
+
+######################################################
+# Final reprocessing rules, for all experiments
+
+rule copy_experiment_from_analysis_to_atlas_exps:
+    """
+    Copy data to atlas_exps. Its execution timing depends on experiment_type.
+    NOTE: Use target_dir = config[atlas_exps] for production
+    """
+    conda: "envs/perl-atlas-modules.yaml"
+    log: "logs/{accession}-copy_experiment_from_analysis_to_atlas_exps.log"
+    input: get_checkpoints_cp_atlas_exps
+    params:
+        target_dir=get_tmp_dir()
+    output:
+        temp("logs/{accession}-copy_experiment_from_analysis_to_atlas_exps.done")
+    shell:
+        """
+        set -e # snakemake on the cluster doesn't stop on error when --keep-going is set
+        exec &> "{log}"
+        export ATLAS_EXPS={params.target_dir}"/tmp" # {params.target_dir} for production
+        source {workflow.basedir}/bin/reprocessing_routines.sh
+
+        echo "Copying data to stage for: {wildcards.accession}"
+
+        copy_experiment_from_analysis_to_atlas_exps {wildcards.accession}
+
+        echo "Copied data to stage"
+
+        touch {output} 
+        """
+
+
+rule get_magetab_for_experiment:
+    """
+    Generate condensed SDRF with Zooma mappings for the experiment - in atlas_exps.
+    NOTE: Use target_dir = config[atlas_exps] for production
+    """
+    conda: "envs/perl-atlas-modules.yaml"
+    log: "logs/{accession}-get_magetab_for_experiment.log"
+    input: rules.copy_experiment_from_analysis_to_atlas_exps.output
+    params:
+        target_dir=get_tmp_dir(),
+        exp_type=get_from_config_or_metadata_summary('experiment_type'),
+        zooma_exclusions=get_zooma_exclusions()
+    output:
+        temp("logs/{accession}-get_magetab_for_experiment.done")
+    shell:
+        """
+        set -e # snakemake on the cluster doesn't stop on error when --keep-going is set
+        exec &> "{log}"
+        export ATLAS_EXPS={params.target_dir}"/tmp"  # edit {params.target_dir} for production
+        source {workflow.basedir}/bin/reprocessing_routines.sh
+        # atlas_env -> 'prod' in bin/reprocessing_routines.sh
+
+        echo "Retrieving magetab files for {wildcards.accession}"
+
+        get_magetab_for_experiment {wildcards.accession} {params.exp_type} {workflow.basedir} {params.zooma_exclusions}
+        
+        echo "Retrieved magetab files for {wildcards.accession}"
+
+        touch {output} 
+        """
+
+######################################################
+# Potential new rule, to be implemented
+
+rule delete_experiment:
+    """
+    Delete experiment from Atlas.
+    It should also remove its RUN IDs from ISL db.
+    add goal='delete' in addition to 'recalculations' and 'reprocess'
+    """
```

Comment:
@pcm32 I've noticed this is something done by the curation team under current _Updating existing experiments_ SOP. Could you confirm we should not implement this here?

Reply:
Currently this happens via a Jenkins job, which probably today is using a call to the web application besides some deletions... I'm not sure how we are doing and to what extent is done (solr, db, file system), but we will need a system for this. I suspect that currently the easiest is to continue to rely on the web application call until we devise a web-cli call for this... or have we already?

Reply:
ahhh, ok, no implementation here yet. I would leave it (as in not doing it - remove it if you want) in the meantime. The Jenkins job is surely relying on the web application for this.